src/GraphQL/Conversion/CamelCaseNameConverter.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using GraphQL.Types;
+
+namespace GraphQL.Conversion
+{
+    /// <summary>
+    /// Camel case name converter; set as the default <see cref="INameConverter"/> within <see cref="ExecutionOptions.NameConverter"/>.
+    /// Always used by all introspection fields regardless of the selected <see cref="INameConverter"/>.
```

Comment:
Generally speaking, it would be more correct to say that introspection fields do not use any converter. And so it should be in code.

Reply:
That is presently incorrect.  Due to some of the introspection fields being defined as `Field(x => x.Name)`, they do rely on a camel case converter for proper operation (verified).  I can change this if you want.

Reply:
Also this comment can be written on the `INameConverter` interface.

Reply:
Added; please review.

Reply:
> That is presently incorrect. Due to some of the introspection fields being defined as Field(x => x.Name), they do rely on a camel case converter for proper operation 

You're right.